trading_ig/rest.py

```diff
@@ -74,11 +72,11 @@ def _url(self, endpoint):
         """Returns url from endpoint and base url"""
         return self.BASE_URL + endpoint
 
-    def _create_first(self, endpoint, params, session):
+    def _create_first(self, endpoint, params, session, version):
         """Create first = POST with headers=BASIC_HEADERS"""
         url = self._url(endpoint)
         session = self._get_session(session)
-        params['password'] = params['password'].decode()
+        params['password'] = params['password'] # .decode()
```

Comment:
See https://github.com/ig-python/ig-markets-api-python-library/issues/108#issuecomment-619519234

Reply:
had a look, revised and made the required changes